crowdin/messages.json

```diff
@@ -2106,15 +2104,15 @@
   "-1746837387": "We may have declined your documents because they were unclear, invalid, expired, had cropped edges, or showed details that did not match your Deriv profile. If you need help, please <0>contact us via live chat</0>.",
   "-1248152791": "When you join our affiliate programme,",
   "-1663161989": "You'll have multiple income opportunities by joining other partnership programmes that we offer.",
+  "-110767819": "You'll have access to various marketing tools to help promote our products and services to your clients.",
   "-640475298": "You'll be able to take advantage of our multilingual platforms and reach clients wherever they are.",
   "-179421588": "You'll receive prompt monthly payouts via your preferred payment method.",
   "-874599648": "You won't be charged any hidden fees. You'll be able to see exactly what your earnings are.",
   "-1456353153": "You'll have a dedicated account manager to help you.",
   "-1835477277": "<0>Cost per acquisition</0> (For the EU only)",
   "-2057848186": "Earn 100 USD in commission for each successful referral. Your referred client must open a real account using your tracking link and deposit 100 USD or more (one-time or accumulative).",
   "-1264084852": "We encourage you to become our affiliate if you are:",
-  "-317355829": "<0>A webmaster</0><br/>\n                Do you have a trading-related website? Join our network of affiliates and turn your traffic into revenue by promoting our products and services.",
-  "-643644414": "<0>A trading mentor</0><br/>\n                Do you mentor other traders? Help them become better traders and earn commission when they sign up and trade on our platforms.",
```

Comment:
we don't use this anywhere? Looks like the content is not added back anywhere.

Reply:
These keys contain `\n` which is wrong ( in both old and new approach ), all of these will be resolved in [this PR](https://github.com/binary-com/deriv-com/pull/3564)